scripts/write_jsons.py

```diff
@@ -102,14 +106,35 @@ def get_hh(fault_dir):
     :return: hh value string
     """
     os.chdir(fault_dir)
-    params_base = PARAMS_BASE
-    if os.path.isfile(params_base):
+    if os.path.isfile(PARAMS_BASE):
         cmd = "grep 'hh =' params_base.py"
         output = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True).communicate()[0]  # output = "hh = '0.4' #must be in formate like 0.200 (3decimal places)\n"
         hh = output.split("#")[0].split("=")[1].strip().replace("'", '')  # "'0.4'" ---> '0.4'
         return hh
 
 
+def get_submit_time(ch_log_dir, sim_type, realization):
+    """
+    get submit_time for a realization by reading its log file
+    :param sim_type: [BB|HF|LF]
+    :param realization: eg.Hossack_HYP222-501_S3454
+    :return: submit_time
+    """
+    if sim_type == 'LF':
+        sim_type = 'EMOD3D'
+    time_file = os.path.join(ch_log_dir, "{}.{}.log".format(sim_type, realization))
+    if os.path.isfile(time_file):
+        with open(time_file, 'r') as f:
+            submit_time = f.read().split(':')[-1].strip()  # 20181031_060458
+        date, time = submit_time.split('_')
+        try:
+            return "{}-{}-{}_{}:{}:{}".format(date[:4], date[4:6], date[6:], time[:2], time[2:4], time[4:])  # 2018-10-31_06:59:22
```

Comment:
Any reason why you wouldn't use `datetime.strptime` to parse the date and then `datetime.strftime` to format it after?

Reply:
Good point Thanks a lot!